azure-pipelines.yml

```diff
@@ -182,6 +182,45 @@ jobs:
       includePatterns:
         - "/sw/device/**/*nexysvideo*"
 
+# Software targeting the English Breakfast top level is produced by patching
+# the source tree before building. This builds only the boot ROM and Hello World
+# example.
```

Comment:
Also the AES serial application? I'd just reduce this comment to "This only builds a select subset of software."

Reply:
Done.